### DISCUSSION TITLE
Request - *Arr Suite LXC

### DISCUSSION THREAD

**Discussion by ifttunes** [tteck/Proxmox#1860] [Ideas]
It's possible to create a larger LXC for all arr suite? 
I found a little bit frustrating the need to create individual LXC container for each *arr, and mapping my smb shares to each one.
Should be nice to have a full suite install, even better if i can select the apps that i need, like sonarr, radarr, prowlarr, bazarr, and even a torrent client (this one I get if you choose to not include) but i think all the others would benefit sharing resources and mount points

**Comment by jfamiens** [comment]
@ifttunes  I concur, struggle right now with permissions to enhance my experience with these  *arr containers.

**Comment by yodaphone** [comment]
I found a script maybe this might be a starting point?

https://wiki.servarr.com/install-script

Or this one based on a VM (TrueNAS) +Docker, with Windscribe built right in

https://github.com/imjustleaving/trueNAS/wiki/TrueNAS---Media-Server-Virtual-Machine
